pages/mentoring_workflow.md

```diff
@@ -42,6 +42,8 @@ Once you're happy with the feedback you've written, click "Comment". If you're n
 
 If you come across a solution you don't want to give feedback to, you can click the "I'll pass, thanks" button and you won't ever see the solution in your Mentor Dashboard queue again.
 
+_Note: this button is labeled as `Leave conversation` for solutions you are actively mentoring.  Clicking this button will return the solution to the mentor queue for another mentor to assume._
+
```

Comment:
If you made a comment on the submission, will it allow you to still track the conversation?  Or does it disallow the notifications after this point?

Reply:
Good question.  I need @iHiD to look this over when he gets a chance.

Reply:
If you leave, you leave. No more notifications etc.

Reply:
There might be / have been a bug there --- I have left a solution and up till 2 days ago I did still receive notifications whenever the student _or_ the new mentor responded.

Reply:
@SleeplessByte Open an issue at exercism/exercism for me to double-check this?

Reply:
https://github.com/exercism/exercism/issues/4700